keyboards/ut472/keymaps/praxling_dvorak/rules.mk

```diff
@@ -0,0 +1,34 @@
+# MCU name
+MCU = atmega32u2
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = atmel-dfu
+
```

Comment:
`rules.mk` at the keymap level should only contain overrides, so at a minimum MCU and BOOTLOADER should be removed.